src/main/java/mate/academy/shop/anotation/Injector.java

```diff
@@ -9,11 +12,13 @@
 import java.util.List;
 
 public class Injector {
+    final static Logger logger = Logger.getLogger(FileReader.class);
     private static final String PROJECT_MAIN_PACKAGE = "mate.academy.shop";
     private static List<Class> classes = new ArrayList<>();
 
     static {
         try {
+            logger.info("Injector start working");
             classes.addAll(getClasses(PROJECT_MAIN_PACKAGE));
         } catch (ClassNotFoundException | IOException e) {
             e.printStackTrace();
```

Comment:
remove e.printStackTrace(); and use logger

Reply:
done